xarray/tests/test_units.py

```diff
@@ -2237,6 +2243,21 @@ def test_pad_unit_constant_value(self, unit, error, dtype):
         assert_units_equal(expected, actual)
         assert_identical(expected, actual)
 
+    @requires_dask
```

Comment:
these tests are about `dask` and duck dask arrays (`pint` is just an example of a duck dask array), so I think it makes more sense to move them to `test_dask.py`

Reply:
Okay, that makes sense. What is the best way to import and set up `pint` in that case? A `pytest.importorskip("pint")` wouldn't be good. I'm thinking a guarded import that contains `pint` related setup, then a `skipif` for `pint` < 0.15.

Reply:
you could define a `requires_pint` decorator in `xarray/tests/__init__.py`. For the setup, I guess it's fine to add:
```python
import pint
unit_registry = pint.UnitRegistry(force_ndarray_like=True)
```
at the top of both test functions.